Added parameters autoselection to docs (#40408)
ydb-platform/ydb: ydb/docs/en/core/yql/reference/syntax/alter_table/indexes.md

{% include [vector_index_parameters.md](../_includes/vector_index_parameters.md) %}

{% note info %}

> For vector indexes we can omit vector_type and vector_dimension parameters if table is not empty. They will be autodetected from the row contents. Parameters levels and clusters are also can be autodetected, but it doesn't requires non empty table.

Reviewer: ```suggestion
For vector indexes, the `vector_type` and `vector_dimension` parameters can be omitted if the table is not empty — they will be autodetected from the row contents. The `levels` and `clusters` parameters can also be autodetected, and they do not require a non-empty table.
```
